src/parallel/zipmacro.rs

```diff
@@ -47,60 +49,13 @@
 ///     // Compute a simple ternary operation:
 ///     // elementwise addition of b and c, stored in a
 ///
-///     par_azip!(mut a, b, c in { *a = b + c });
+///     par_azip!((a in &mut a, &b in &b, &c in &c) *a = b + c);
 ///
 ///     assert_eq!(a, &b + &c);
 /// }
 /// ```
 macro_rules! par_azip {
-    // Build Zip Rule (index)
-    (@parse [index => $a:expr, $($aa:expr,)*] $t1:tt in $t2:tt) => {
-        $crate::par_azip!(@finish ($crate::Zip::indexed($a)) [$($aa,)*] $t1 in $t2)
-    };
-    // Build Zip Rule (no index)
-    (@parse [$a:expr, $($aa:expr,)*] $t1:tt in $t2:tt) => {
-        $crate::par_azip!(@finish ($crate::Zip::from($a)) [$($aa,)*] $t1 in $t2)
-    };
-    // Build Finish Rule (both)
-    (@finish ($z:expr) [$($aa:expr,)*] [$($p:pat,)+] in { $($t:tt)*}) => {
-        use $crate::parallel::prelude::*;
```

Comment:
Didn't notice it before, but `par_apply` is an inherent method nowadays and no trait use is needed.